File: BeeCrowd/Pomekons.java
// 2174 - Coleção de Pomekons (05/02/2025)
import java.util.*;

public class Pomekons {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int quantidade = sc.nextInt();

        String pomekons[] = new String[quantidade];
        for(int i = 0; i < quantidade; i++){
            pomekons[i] = sc.next();
        }

        int contador = 0, colecionados = 0;;

        for(int i = 0; i < quantidade; i++){
            boolean unico = true;
            for(int j = 0; j < i; j++){
                if(pomekons[i].equals(pomekons[j])){
                    unico = false;
                    break;
                }
            }
            if(unico){
                contador++;
            }
        }

        colecionados = 151 - contador;

        System.out.print("Falta(m) " + colecionados + " pomekon(s).");

        sc.close();
    }
}

File: BeeCrowd/Pomekons.c
// 2174 - Coleção de Pomekons (05/02/2025)
#include <stdio.h>
#include <string.h>
#include <stdbool.h>

int main(){
    int quantidade, contador = 0, coletados = 0;
    scanf("%d", &quantidade);
    getchar();

    char pomekons[quantidade][100];

    for (int i = 0; i < quantidade; i++){
        fgets(pomekons[i], 100, stdin);
        pomekons[i][strcspn(pomekons[i], "\n")] = 0; // Para remover a quebra de linha e ficar \0
    }

    for (int i = 0; i < quantidade; i++){
        bool unico = true;
        for (int j = 0; j < i; j++){
            if (strcmp(pomekons[i], pomekons[j]) == 0) {
                unico = false;
                break;
            }
        }
        if (unico){
            contador++;
        }
    }

    coletados = 151 - contador;

    printf("Falta(m) %d pomekon(s). ", coletados);

    return 0;
}